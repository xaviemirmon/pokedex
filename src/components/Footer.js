/** @jsx jsx */

// Package imports
import { Link, jsx } from "theme-ui"

const Footer = () => (
  <footer
    sx={{
      mx: `auto`,
      textAlign: `center`,
      my: 4,
    }}
  >
    Built with
    {` `}
    <Link href="https://www.gatsbyjs.com">Gatsby</Link>
    {` `}
    using data from{" "}
    <Link href="https://graphql-pokemon2.vercel.app">
      graphql-pokemon2.vercel.app
    </Link>
    . All images are copyright to their respective owners.
  </footer>
)

export default Footer
