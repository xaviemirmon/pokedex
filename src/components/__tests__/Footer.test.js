// Package imports
import React from "react"
import renderer from "react-test-renderer"

// Component imports
import Footer from "../Footer"

describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
