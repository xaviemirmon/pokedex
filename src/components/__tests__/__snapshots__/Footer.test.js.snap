// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="css-cjzi5r-Footer"
>
  Built with
   
  <a
    className="css-vurnku"
    href="https://www.gatsbyjs.com"
  >
    Gatsby
  </a>
   
  using data from
   
  <a
    className="css-vurnku"
    href="https://graphql-pokemon2.vercel.app"
  >
    graphql-pokemon2.vercel.app
  </a>
  . All images are copyright to their respective owners.
</footer>
`;
